ladybug_charts/_schema.py

```diff
@@ -0,0 +1,648 @@
+container_col_center_one_of_three = (
+    "container-col justify-center one-of-three-container"
+)
+
+# Misc
+month_lst = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec",
+]
+
+
+clima_template = "plotly_white"
+
+pio.templates.default = clima_template
+template = clima_template
+
+fig_config = {
+    "modeBarButtonsToRemove": [
+        "toggleSpikelines",
+        "hoverCompareCartesian",
+        "select2d",
+        "zoom2d",
+        "autoScale2d",
+        "pan2d",
+        "lasso2d",
+        "zoomIn2d",
+        "zoomOut2d",
+        "hoverClosestCartesian",
+    ],
+    "displaylogo": False,
+    "displayModeBar": "hover",
+    "toImageButtonOptions": {"format": "svg", "filename": "Clima tool graph"},
+}
+
+tight_margins = dict(l=20, r=20, t=33, b=20)
+
+# Units Dictionary
+degrees_unit = "\u00B0deg"
+temperature_unit = "\u00B0C"
+radiation_unit = "Wh/m<sup>2</sup>"
+thermal_stress_label = "Thermal stress"
+
+mapping_dictionary = {
+    "None": {"name": "None"},
+    "DOY": {"name": "Day of the year", "unit": "days", "range": [0, 365]},
+    "day": {"name": "day", "range": [1, 31]},
+    "month": {"name": "months", "unit": "months", "range": [1, 12]},
+    "hour": {
+        "name": "hours of the day",
+        "color": [
+            "#000000",
+            "#355e7e",
+            "#6b5c7b",
+            "#c06c84",
+            "#f8b195",
+            "#c92a42",
+            "#c92a42",
+            "#c92a42",
+            "#000000",
+        ],
+        "unit": "h",
+        "range": [1, 24],
+    },
+    "DBT": {
+        "name": "Dry bulb temperature",
+        "color": ["#00b3ff", "#000082", "#ff0000", "#ffff00"],
+        "unit": "°C",
+        "range": [-40, 50],
+    },
+    "DPT": {
+        "name": "Dew point temperature",
+        "color": ["#00b3ff", "#000082", "#ff0000", "#ffff00"],
+        "unit": "°C",
+        "range": [-50, 35],
+    },
+    "RH": {
+        "name": "Relative humidity",
+        "color": ["#ffe600", "#00c8ff", "#0000ff"],
+        "unit": "%",
+        "range": [0, 100],
+    },
+    "p_atm": {
+        "name": "Atmospheric pressure",
+        "color": [
+            "#ffffff",
+            "#b2f2ff",
+            "#33ddff",
+            "#00aaff",
+            "#0055ff",
+            "#0000ff",
+            "#aa00ff",
+            "#ff00ff",
+            "#cc0000",
+            "#ffaa00",
+        ],
+        "unit": "Pa",
+        "range": [95000, 105000],
+    },
+    "extr_hor_rad": {
+        "name": "Extraterrestrial horizontal irradiation",
+        "color": [
+            "#293a59",
+            "#960c2c",
+            "#ff0000",
+            "#ff7b00",
+            "#fffc00",
+            "#ffff7b",
+            "#ffffff",
+        ],
+        "unit": radiation_unit,
+        "range": [0, 1200],
+    },
+    "hor_ir_rad": {
+        "name": "´Horizontal infrared radiation",
```

Comment:
I sent a PR to fix it on their side but we should also fix it here:

```suggestion
        "name": "Horizontal infrared radiation",
```